lua/nvim-treesitter/config.lua

```diff
@@ -1,3 +1,5 @@
+local util = require('nvim-treesitter.util')
```

Comment:
Not crazy about having to `require` something in the main entry point module; maybe we can move `throttle_by_id` to here? `install` already requires `config`.

(Also not crazy about this sort of module golfing, but keeping `config` light was one of my original motivations for the rewrite...)